assignments/assignments.go

```diff
@@ -26,16 +26,19 @@ func UpdateFromTestsRepo(logger *zap.SugaredLogger, db database.Database, repo *
 		logger.Errorf("Failed to fetch assignments from '%s' repository: %w", pb.TestsRepo, err)
 		return
 	}
+
```

Comment:
Don't think we need to add line breaks here; the single `}` is already quite an empty line, so I don't feel that adding another blank line here helps code comprehension.

Reply:
Done.